WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -11,11 +11,11 @@ extension PostEditor {
             settingsViewController = PostSettingsViewController(post: post)
         }
         settingsViewController.featuredImageDelegate = self as? FeaturedImageDelegate
-        let closeButton = UIBarButtonItem(systemItem: .close, primaryAction: .init(handler: { [weak self] _ in
+        let doneButton = UIBarButtonItem(systemItem: .done, primaryAction: .init(handler: { [weak self] _ in
```

Comment:
It's just a small update. I previously used a "Close" button after making this screen modal, but I felt it wasn't great.